app/static/css/slcQuestionnaire.css

```diff
@@ -0,0 +1,8 @@
+.slcQuestionWordCounter{
+    font-size: 0.8em;
+    color: gray;
+}
+.slcQuestionWordCount{
```

Comment:
the element with this class is inside of the other one, so the font-size and color are inherited and you don't need to specify them.

Reply:
resolved